pages/design/definition-of-done.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Definition of "Done"
+---
+
+## What is this? 
+
+It is important that we in the Auto-WCAG Community Group have a shared understanding of what it means for work to be complete.  The Definition of "Done" can be used to assess when work is complete. ([The Scrum Guide](https://www.scrumguides.org/scrum-guide.html#artifact-transparency-done).
+
+It can be used by rule authors as well as reviewers as a checklist to assess whether a rule is done or if it needs another iteration.
+
+Sticking to the Definition of "Done" ensures a high quality in rules published by the Auto-WCAG Community Group, and ensures transparency in our work. 
+
+The Definition of "Done" is a living document, and might change as the rule writing community develops and matures.
+
+## The Definition of "Done"
+- The relation between the rule and its accessibility requirements is as described in the [ACT Rules Format](https://www.w3.org/TR/act-rules-format/) under [Accessibility Requirements](https://www.w3.org/TR/act-rules-format/#structure-accessibility-requirements). This includes:
+    - The rule is an appropriate test for the accessibility requirement it sets out to test, e.g. from [WCAG 2](https://www.w3.org/TR/WCAG21/), and in line with the supporting documents, e.g. [Understanding WCAG 2.0](https://www.w3.org/TR/UNDERSTANDING-WCAG20/) and [Techniques for WCAG 2.0](https://www.w3.org/TR/WCAG-TECHS/)
+    - Failing the rule will for sure mean failing the accessibility requirements (success criteria) linked to under the "Success criteria" heading. If this is not true, the success criteria can instead be mentioned under the heading "Background".
```

Comment:
This is not quite in line with the current formulation in ACT RF:

> Outcomes from an ACT Rule SHOULD be consistent with the accessibility requirement, e.g. a rule only returns the outcome Fail when the content fails the accessibility requirement.

This might be something we should adjust in ACT RF though?

Reply:
I hope someone is working on that adjustment. It was up a few weeks ago splitting it out, so that there were MUST requirements for the specific rule types.